quickstarts/prometheus-agent/config.yml

```diff
@@ -0,0 +1,38 @@
+slug: prometheus-agent
+
+description: |
+  ## About New Relic Prometheus Agent
+  
+  New Relic created the [Prometheus Configurator](https://github.com/newrelic/newrelic-prometheus-configurator) in order to simplify the experience of configuring features like discovery, filtering, metrics decoration, and sharding. 
```

Comment:
I would replace Configurator by Agent even knowing we're referencing the repo itself, otherwise it could be confusing. In the previous line we named it as Agent

Reply:
Updated here dc417d4add7042ba507b7f98f7d123c67cd2f44b